src/utils.js

```diff
@@ -115,7 +115,9 @@ function addToSet(set, array, transformCaseFunc = stringToLowerCase) {
  */
 function cleanArray(array) {
   for (let index = 0; index < array.length; index++) {
-    if (getOwnPropertyDescriptor(array, index) === undefined) {
+    const isPropertyExist = Object.prototype.hasOwnProperty.call(array, index);
```

Comment:
JavaScript does not protect the property name hasOwnProperty, so objects with properties with this name may return incorrect results.

I think this is a stable way to use it.

#### Related
[MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwnProperty#using_hasownproperty_as_a_property_name)
[eslint no-prototype-builtins](https://eslint.org/docs/latest/rules/no-prototype-builtins)